crates/holochain_sqlite/tests/tests/schema.rs

```diff
@@ -0,0 +1,81 @@
+use holo_hash::{AgentPubKey, DnaHash};
+use holochain_sqlite::db::{
+    DbKindAuthored, DbKindCache, DbKindConductor, DbKindDht, DbKindP2pAgents, DbKindP2pMetrics,
+    DbKindT, DbWrite,
+};
+use holochain_sqlite::error::DatabaseResult;
+use holochain_sqlite::prelude::DbKindWasm;
+use holochain_zome_types::cell::CellId;
+use kitsune_p2p_bin_data::{KitsuneBinType, KitsuneSpace};
+use std::sync::Arc;
+use walkdir::WalkDir;
+
+#[tokio::test(flavor = "multi_thread")]
+async fn check_schema_migrations_execute() {
```

Comment:
:+1:

Reply:
Would have helped me out, I forgot to `include_str` my new migration and wondered why I was getting so many sql errors. Now I can fail quicker :D